index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title></title>
+	<link rel="stylesheet" href="quiz.css">
+</head>
+<body>
+	<article class="product">
+	  <section class="s1">
+		<header id="header" class="">
+			<h1>Headphones</h1>
+		</header>
+		<div class="image">
+			<img src="http://www.5kmx.com/productimage/2015-12-14-06-22-59_pxc_450_productimage.jpg" alt="">
+		</div>
+		<div class="description">
+			<p>Bacon ipsum dolor amet flank beef shank sirloin ball tip, biltong pork chop cow tenderloin bacon short ribs meatloaf short loin chuck. Kevin tenderloin meatball, pastrami turkey corned beef kielbasa filet mignon pork belly ball tip rump beef.
+			</p>
+		</div>
+		<div class="availability">
+			<span class="available"><p>Available</p></span>
+		</div>
+	  </section>
+	  <section class="s2">
+	  	<header>
+			Specs
```

Comment:
**Requirement missed:** This section should have a header containing the word "Specifications"